services/jupyterhub/spawners.py

```diff
@@ -263,26 +264,31 @@ def get_pod_manifest(self):
                 container for container in self.singleuser_init_containers
                 if not container.name.startswith('renga-')
             ]
-            self.singleuser_init_containers.append({
-                'name':
-                    container_name,
-                'image':
-                    'alpine/git',
-                'command': ['sh', '-c'],
-                'args': [
-                    'git clone {repository} {mount_path} && '
-                    '(git checkout {branch} || git checkout -b {branch}) && '.
-                    format(
-                        branch=options.get('branch'),
-                        commit_sha=options.get('commit_sha'),
-                        mount_path=mount_path,
-                        repository=repository,
-                    )
-                ],
-                'volumeMounts': [volume_mount],
-                'workingDir':
-                    mount_path,
-            })
+            self.singleuser_init_containers.append(
+                client.V1Container(
+                    **{
+                        'name':
+                            container_name,
+                        'image':
+                            'alpine/git',
+                        'command': ['sh', '-c'],
+                        'args': [
+                            'git clone {repository} {mount_path} && '
+                            '(git checkout {branch} || '
+                            'git checkout -b {branch}) && '
```

Comment:
you should keep the ||'ed commands on one line for clarity

Reply:
Ideally, yes but it makes the line too long

Reply:
Indent then